GUIDELINES.md

```diff
@@ -4,121 +4,121 @@ General guidelines about how to successfully develop an application based on [co
 Controllers should do all HTTP related stuff.
 
 #####Take care of
-- HTTP requests and responses.
-- Cookies and sessions.
-- Forms.
-- Context and other Play lifecycle specific stuff.
-- Templates.
+* HTTP requests and responses.
+* Cookies and sessions.
+* Forms.
+* Context and other Play lifecycle specific stuff.
+* Templates.
 
 #####Patterns and rules
-- Dependency injection of services.
-- Simple code.
-- Should end with the word "Controller".
-- Most likely does not have mutable state.
-- Separate controller classes according to login required or not, then use controller-wide annotations.
+* Dependency injection of services.
+* Simple code.
+* Should end with the word "Controller".
+* Most likely does not have mutable state.
+* Separate controller classes according to login required or not, then use controller-wide annotations.
 
 #####How do tests look like
-- Check cookies and sessions.
-- Check response code.
-- Check certain words in response content.
-- Use `callAction` with `fakeRequest` from Play.
-- Extend test classes like `WithApplication` instead of creating an anonymous runnable in every test method.
+* Check cookies and sessions.
+* Check response code.
+* Check certain words in response content.
+* Use `callAction` with `fakeRequest` from Play.
+* Extend test classes like `WithApplication` instead of creating an anonymous runnable in every test method.
 
 ###Services
 Services request domain related stuff, usually to external services.
 
 #####Take care of
-- Sphere and any other service needed.
-- Communication with external systems.
-- No access to HTML templates, except for email templates.
+* Sphere and any other service needed.
+* Communication with external systems.
+* No access to HTML templates, except for email templates.
 
 #####Patterns and rules
-- Asynchronous calls for external services.
-- Interface with implementation to mock external result.
-- May be reused.
-- May end with with the term "Service".
-- Should be thread-safe.
-- Most likely doesn't have mutable members.
+* Asynchronous calls for external services.
+* Interface with implementation to mock external result.
+* May be reused.
+* May end with with the term "Service".
+* Should be thread-safe.
+* Most likely doesn't have mutable members.
 
 #####How do tests look like
-- Should never use running application.
-- Use mocking for external system.
+* Should never use running application.
+* Use mocking for external system.
 
 ###Models
 Models hold local data containers and domain logic.
 
 #####Take care of
-- Only aware of other models.
+* Only aware of other models.
 
 #####Patterns and rules
-- Immutable objects.
-- Highly reused.
-- Created by static factory methods or builders.
-- Should not end with the term "Model".
+* Immutable objects.
```

Comment:
Yes, that's why we have to put them up-to-date. I just fixed formatting and barely nothing else. I'd say, let's just ignore it and fix it in this issue: https://github.com/commercetools/commercetools-sunrise-java/issues/450